controllers/login.js

```diff
@@ -41,11 +41,16 @@ router.post('/login/', (req, res, next) => {
 				},
 				authHelper.cookieDomain(res)));
 		res.redirect('/login/success/');
-	}).catch(() => {
+	}).catch((e) => {
 		res.locals.notification = {
 			type: 'danger',
 			message: 'Login fehlgeschlagen.',
+			statusCode: e.statusCode,
+			timeToWait: process.env.LOGIN_BLOCK_TIME || 15
```

Comment:
Could you also add the env variable `LOGIN_BLOCK_TIME=3` or something small to the testrun dockerfiles so they don't need to wait that long?
https://github.com/schul-cloud/docker-compose/blob/develop/docker-compose.integration-test-Build.yml#L67

Reply:
Is there a test which do a wrong login attempt? Otherwise for successful logins it won't block the login.

Reply:
Yes there is. We have a test that changes the user password and then checks if the user can still login with the old password.

Reply:
I made it dependent now from die env variable in integration test